index.md

```diff
@@ -41,9 +44,9 @@ and a continously updating leaderboard. For code examples and how to contribute,
 2. 10K CHF
 3. 5K CHF
 
-The 3rd prize is reserved for the top contribution from developing countries (see [Table C p.166](https://www.un.org/development/desa/dpad/wp-content/uploads/sites/45/WESP2020_Annex.pdf) for country list). If such a contribution is already among the top 2, any thrid contribution will get the 3rd prize.
+The 3rd prize is reserved for the top contribution from developing countries (see [Table C p.166](https://www.un.org/development/desa/dpad/wp-content/uploads/sites/45/WESP2020_Annex.pdf) for country list). If such a contribution is already among the Top 2, any thrid contribution will get the 3rd prize.
 
-The organizers thank WWRP for providing the money of the prize. missing some org here?
+The organizers thank WWRP for providing the money of the prize.
```

Comment:
should we give credit to the organizations funding the prize? wasnt it two orgs?